index.rst

```diff
@@ -70,292 +70,140 @@ In gross, the conceptual steps are:
-
-https://ci.lsst.codes/blue/organizations/jenkins/release%2Frun-rebuild/activity
-
-The resulting manifest ID needs to be retrieve to use as input for subsequent
-jobs.
+1st Release Candidate
+---------------------
 
-.. code-block::
+Build and Publish
+^^^^^^^^^^^^^^^^^
 
-    REFS: v42.0.0.rc1
-    PRODUCTS: lsst_distrib
-    BUILD_DOCS: true
+Run the jenkins `release/official-release <official-release>`_ job.  The source
+git refs should be only the tag of the "seed" weekly release.  The release tag
+**must** start with a ``v`` and end with ``.rc1``.
 
-https://ci.lsst.codes/blue/organizations/jenkins/release%2Frun-publish/activity
+See git-tags_ for details on the formatting of git tags.
 
-.. code-block::
+Example:
 
-    PRODUCTS: lsst_distrib
-    EUPSPKG_SOURCE: git
-    TAG: v42_0_0_rc1
-    BUILD_ID: bXXXX
+.. code-block:: text
 
-Build and Publish eups ``tarball`` packages
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   SOURCE_GIT_REFS: w.1999.42
+   RELEASE_GIT_TAG: v42.0.0.rc1
+   O_LATEST: false
 
-Note that the jenkins ``release/official-release`` job does not trigger
-``release/tarball-matrix`` and triggers ``release/tarball`` build(s) directly
-so as to have more explicit control over the parameters.
-
-https://ci.lsst.codes/blue/organizations/jenkins/release%2Ftarball-matrix/activity
+Announce rc1
+^^^^^^^^^^^^
 
-.. code-block::
+Update the release status `community.lsst.org <clo>`_ post to to announce the
+availability of ``rc1``.
 
-    PRODUCTS: lsst_distrib
-    EUPS_TAG: v42_0_0_rc1
-    SMOKE: true
-    RUN_SCONS_CHECK: true
-    PUBLISH: true
 
-Build and Publish ``scipipe`` docker image
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+2nd+ Release Candidate(s)
+-------------------------
 
-https://ci.lsst.codes/blue/organizations/jenkins/release%2Fdocker%2Fbuild-stack/activity
+An ``.rcX``, where X is ``> 1``, is only required if a problem is found with
+the initial ``rc``.
 
-.. code-block::
+**Any subsequent ``rc` differs slightly from the initial ``rc1`` process
+because it inherently is not identical to a previous ``git tag`` (if it was,
+there would be no reason to produce another ``rc``). The creation of a git
+release branch prior to ``rc1`` would eliminate the differences.**
 
-    PRODUCTS: lsst_distrib
-    TAG: v42_0_0_rc1  # eups tag
-    NO_PUSH: false
+Branch, Merge, Tag
+^^^^^^^^^^^^^^^^^^
 
-Build and Publish ``jupyterlab`` docker image
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Any git repository that needs to be modified for additional ``rc`` releases
+should be **branched** and have the necessary changes merged to a release
+branch.  Eg., if changes were needed in ``v42.0.0.rc1`` a "release branch"
+along the lines of ``v42.x`` should be created in the repos that need changes.
 
-https://ci.lsst.codes/blue/organizations/jenkins/sqre%2Finfra%2Fbuild-jupyterlabdemo/activity
+(**TBD**: merge to master and cherry-pick to release branch or merge to release
```

Comment:
This need to be updated in the dev-guide.
Assuming that all fixes and new functionalities are developed in ticket branches, and by default there is a pull request to master, shall we suggest to have an additional pull request to the release branch?

Reply:
The same source branch will only work if there aren't merge conflicts.  I lean slightly towards merge to master and then cherry-picking as it ensure the fix is *always* on master.

Reply:
I think the convention, be it forward-port or back-port, should be RFC'd since it for anything non-trivial, sci-pipe devs should be doing the merges so they are responsible for resolving conflicts.